generic_expression_patterns_modules/download_recount2_data.R

```diff
@@ -60,7 +60,7 @@ get_recount2_sra_subset <- function(template_project_id,
   metadata <- all_metadata()
   write.table(
     metadata,
-    paste(base_dir, '/human_analysis/data/metadata/recount2_metadata.tsv', sep = ""),
+    paste(base_dir, '/human_general_analysis/data/metadata/recount2_metadata.tsv', sep = ""),
```

Comment:
If you want, you can use:
```paste0(base_dir, '/human_general_analysis/data/metadata/recount2_metadata.tsv')```
to simply the code a little bit.

Reply:
Turns out that the default `sep` is a space. So I need to have sep either way so I kept `sep=/`

Reply:
The function that I suggested was `paste0`, not `paste`. But it's no big deal.